docs/outlook/debug-ui-less.md

```diff
@@ -1,16 +1,19 @@
 ---
-title: Debug your UI-less Outlook add-in
-description: Learn how to debug your UI-less Outlook add-in.
+title: Debug function commands in Outlook add-ins
+description: Learn how to Debug function commands in Outlook add-ins.
```

Comment:
```suggestion
description: Learn how to use Debug function commands in Outlook add-ins.
```
Or lower case "debug" if it's a verb and not a command.

Reply:
```suggestion
description: Learn how to debug function commands in Outlook add-ins.
```